test.rb

```diff
@@ -0,0 +1,26 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+aaa;
+
+not 1
```

Comment:
__[development][rubocop]__
* Use `!` instead of `not`. [:ref](https://github.com/bbatsov/ruby-style-guide#bang-not-not)

<!-- Comment by [SideCI](https://sideci.com) -->